services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/NodeService.java

```diff
@@ -26,4 +26,8 @@
     NodeInfo updateNodeInfo(String nodeId, NodeInfo nodeInfo) throws ParameterNullOrEmptyException, InvalidDataException, Exception;
```

Comment:
@pkommoju Is that possible for a node migrating to another ncm?

Reply:
We haven't thought about it, but I think it could be supported by a delete in one NCM followed by add to the other NCM. I am not sure how this would affect DPM and NMM. If there is no strong enough reason, we might be better off to not support it, at least at this time.